platform/ios/src/MGLMapView.h

```diff
@@ -296,6 +296,17 @@ MGL_EXPORT IB_DESIGNABLE
 
 #pragma mark Displaying the User’s Location
 
+/**
+ The object that this map view uses to start and stop the delivery of location-related
+ events.
+ 
+ If no custom manager is provided or setting this property to `nil` will default
+ to the internal `CLLocationManager` based implementation.
```

Comment:
If the default use of CLLocationManager is an internal implementation detail, then we shouldn’t mention it.

Reply:
How can we communicate this behavior? We should let devs know that this is a specialized API and if nothing is provided then the default option is used.

Reply:
We can certainly say that a typical implementation comes with MGLMapView by default, but if the goal is to reduce dependency on CLLocationManager, then we need to couple any mention of CLLocationManager in the documentation with a note that the default location manager may change in the future.